### PR TITLE
[14.0][MIG] project_category

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
LGTM

File: project_category/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* Go to **Project > Configuration > Project Type** and add some items in order to
```
Comment (bodedra) [inline project_category/readme/USAGE.rst]:
```suggestion
* Go to **Project > Configuration >  Project Types** and add some items in order to
```

File: project_category/views/project_project_views.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="project_search" model="ir.ui.view">
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_project_filter" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="type_id" string="Type" />
```
Comment (bodedra) [inline project_category/views/project_project_views.xml]:
```suggestion
                <field name="type_id" />
```

File: project_category/views/project_task_views.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                <field name="type_id" context="{'default_task_ok': True}" />
+            </field>
+        </field>
+    </record>
+    <record id="project_task_search" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_search_form" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="type_id" string="Type" />
```
Comment (bodedra) [inline project_category/views/project_task_views.xml]:
```suggestion
                <field name="type_id" />
```

File: project_category/views/project_type_views.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="project_type_form" model="ir.ui.view">
+        <field name="model">project.type</field>
+        <field name="arch" type="xml">
+            <form string="Type">
```
Comment (bodedra) [inline project_category/views/project_type_views.xml]:
```suggestion
            <form string="Project Type">
```

File: project_category/views/project_type_views.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                <filter
+                    name="task_ok"
+                    string="Can be applied for tasks"
+                    domain="[('task_ok','=',True)]"
+                />
+            </search>
+        </field>
+    </record>
+    <record id="action_project_type" model="ir.actions.act_window">
+        <field name="name">Type</field>
```
Comment (bodedra) [inline project_category/views/project_type_views.xml]:
```suggestion
        <field name="name">Project Types</field>
```

File: project_category/views/project_type_views.xml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        <field name="res_model">project.type</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem
+        name="Project Type"
```
Comment (bodedra) [inline project_category/views/project_type_views.xml]:
```suggestion
        name="Project Types"
```

**Review by bodedra — APPROVED** [review]
Functional reviewed. :+1: 

Added minor suggestion.

**Comment by emagdalenaC2i** [comment]
@bodedra done

**Comment by bodedra** [comment]
@emagdalenaC2i Thank you. Appreciate.

Would you please run pre-commit for make it green again.

**Review by bodedra — APPROVED** [review]

File: project_category/README.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/13.0/project_category
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-13-0/project-13-0-project_category
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/13.0
```
Comment (skeller1) [inline project_category/README.rst:23]: docs need to be regenerated. Still linked to ODOO 13 runbot

File: project_category/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 ADHOC SA (http://www.adhoc.com.ar)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (skeller1) [inline project_category/__manifest__.py]: Is it possible to make all the license links secure (https) http://www.gnu.org/licenses/agpl. We have already converted them in other OCA repositories. Thank you

**Review by skeller1 — CHANGES_REQUESTED** [review]
LGTM, some doc stuff. Left some comments.

File: project_category/README.rst (hunk 7 above)
Comment (emagdalenaC2i) [reply project_category/README.rst:23]: It is created automatically after merge

File: project_category/__manifest__.py (hunk 8 above)
Comment (emagdalenaC2i) [reply project_category/__manifest__.py]: in other OCA repositories?
Please, tell me an example

Comment (skeller1) [reply project_category/__manifest__.py]: https://github.com/OCA/crm/pull/331

Comment (emagdalenaC2i) [reply project_category/__manifest__.py]: ok, done

File: project_category/README.rst (hunk 7 above)
Comment (skeller1) [reply project_category/README.rst:23]: Ohhh, is this a new feature of the 14.0 branch, cool, didn't know that. thank you

**Review by skeller1 — APPROVED** [review]
:+1:

**Comment by emagdalenaC2i** [comment]
@dreispt Are there any issues with the bot?

**Comment by dreispt** [comment]
@emagdalenaC2i Not sure, will try again.

**Pull request merged**